src/app/components/home/home.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="home">
+  <div class="backdrop-home">
+    <div class="container-pokemon-title">
+      <img class="pokemon-title-image" src="../../assets/images/title.png" alt="pokemon-logo" />
+    </div>
+    <div class="welcome"> Welcome to Poketionary! </div>
+    <p class="welcome-description">
+      Know everything you need to know about these awesome creatures
+      of the Pokemon World and choose your best competitor
+    </p>
+    <ngx-slick-carousel class="carousel" #slickModal="slick-carousel" [config]="slideConfig">
+      <div ngxSlickItem *ngFor="let slide of slides" class="slide">
+        <img src="{{ slide.img }}" alt="" width="100%">
```

Comment:
avoid the use of interpolation ( {{}} ) use the input notation instead ( [src]="" ) if there is no input for the attribute use the generic input [attr.atributename]=""